test/app.test.tsx

```diff
@@ -0,0 +1,137 @@
+import { mount, shallow } from 'enzyme'
+import React from 'react'
+import { act } from 'react-dom/test-utils'
+import { Snackbar } from '@material-ui/core'
+import { App } from '../src/app'
+import { RepositoryContext } from '../src/context/repository-provider'
+import { SimpleAppBar } from '../src/components/SimpleAppBar'
+import { AdvancedGridList } from '../src/components/AdvancedGridList'
+import { images } from './mocks/images'
+
+describe('App Layout', () => {
+  it('Matches snapshot', () => {
+    const l = shallow(<App />)
+    expect(l).toMatchSnapshot()
+  })
+  it('getselectedImage with result', async () => {
+    const repository = {
+      loadCollection: () => {
+        return { d: { results: images } }
+      },
+      configuration: {
+        repositoryUrl: 'url',
+      },
+    }
+    let wrapper: any
+    await act(async () => {
+      wrapper = mount(
+        <RepositoryContext.Provider value={repository as any}>
+          <App />
+        </RepositoryContext.Provider>,
+      )
+    })
+    expect(wrapper).toMatchSnapshot()
```

Comment:
This is creating too huge snapshot. It would be better to expect something else.

Reply:
Done